### PR TITLE
Add CI pipeline for Azure Functions project

### PR DESCRIPTION
**Pull request by NT-D**
## Purpose
For keeping high quality code, I want to automate lint, test and build with pipeline

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Author pre-publish checklist:

- [ ] Added test to prove my fix is effective or new feature works
- [x] No PII in logs
- [ ] Made corresponding changes to the documentation

### CODE REVIEW

**Comment by NT-D** [comment]
Add screen shot for validation
image

**Review by jmostella — APPROVED** [review]
LGTM

File: e2e_samples/temperature_events/devops/azure-pipelines-ci-artifacts.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+trigger:
+  branches:
+    include:
+    - master
+    - release/*
```
Comment (jmostella) [inline e2e_samples/temperature_events/devops/azure-pipelines-ci-artifacts.yml:5]: do we intend to have a release branch?

**Review by devlace — APPROVED** [review]
LGTM 👍

**Review by DavidBurela — APPROVED** [review]

**Pull request merged**